### PR TITLE
As an admin, I should be able to edit existing subscription models

### PR DESCRIPTION
**Pull request by omar-mohyeldin**
issue #369 
@AmrARaouf 
@nohahesham 
@Mirna-Yacout 
@karimngr 
Please review

### CODE REVIEW

**Comment by omar-mohyeldin** [comment]
@Mirna-Yacout @AmrARaouf @nohahesham @karimngr please review

**Comment by Mirna-Yacout** [comment]
- the arrow going from view_subscription_models to controller should state which method
- upon clicking ta3deel state which method does the arrow go

File: arability/app/models/subscription_model.rb [hunk 1]
```diff
@@ -1,4 +1,24 @@
+#encoding: UTF-8
 class SubscriptionModel < ActiveRecord::Base
-	has_many :my_subscriptions
+
+  has_many :my_subscriptions
```
Comment (nohahesham) [inline arability/app/models/subscription_model.rb:3]: remove extra empty lines

**Comment by omar-mohyeldin** [comment]
@Mirna-Yacout @nohahesham please re-review

**Comment by Mirna-Yacout** [comment]
verified

**Comment by ghost** [comment]
verified

**Comment by AmrARaouf** [comment]
doc verified

File: arability/spec/controllers/admin_controller_spec.rb [hunk 2]
```diff
@@ -1,21 +1,63 @@
...
-		get :index
-		response.should redirect_to :action => :login
-	end
-
-	it "renders the login template" do
+
+  describe "GET #edit_subscription_model" do
+    let(:model1) do
+      model = SubscriptionModel.new
+      model.name = "Test"
```
Comment (AmrARaouf) [inline arability/spec/controllers/admin_controller_spec.rb]: subscription_model does not have an attribute name, there are are name_ar and name_en only.

There is something wrong in the attributes accessible of the subscription_model.rb, there iare :name and no :name_ar or :name_en, kindly change those as well

**Comment by omar-mohyeldin** [comment]
@AmrARaouf please re-review

**Pull request merged**